torch/csrc/jit/tensorexpr/ir_mutator.cpp

```diff
@@ -282,11 +282,7 @@ const Expr* IRMutator::mutate(const ReduceOp* v) {
   }
 
   return new ReduceOp(
-      buf_new,
-      ExprHandle(body),
-      v->interaction(),
-      new_output_args,
-      new_reduce_args);
+      buf_new, body_new, new_output_args, new_reduce_args, v->reducer());
```

Comment:
Are we recurring into `v->reducer()` at all ?

Reply:
We can't recurse into the reducer since it's not a Node.